tools/azure-sdk-tools/devtools_testutils/azure_recorded_testcase.py

```diff
@@ -37,6 +37,19 @@
 load_dotenv(find_dotenv())
 
 
+def add_sanitizer(sanitizer, regex=None, value=None):
```

Comment:
The `/Admin/AddSanitizer` response is going to be passing back an `id` in the near future. That `id` will be later used by `/Admin/RemoveSanitizer` to remove one at a time if we don't want to have to do a full reset.

Given that, it may be valuable to add the newly POSTED sanitizer to a dictionary that we can import from later.

Reply:
The more I think about it, the more I think we should make the arguments `kwargs` instead of setting them exactly.

That way in each sanitizer, we can access the kwarg that we expect to be present, and throw if it's not.

I say this because right now you're only dealing with configuration that needs a `value` and a `regex`...but what happens if you want to provide a `regexGroup`?

Reply:
That makes total sense -- I had made these optional for simplicity when it was still in `integrate-proxy`, but shifting to kwargs is definitely better. Will do!